listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -251,7 +273,16 @@ export default class ListenCard extends React.Component<
           isCurrentlyPlaying ? "current-listen" : ""
         }${compact ? " compact" : " "} ${className || ""}`}
       >
-        {thumbnail && <div className="listen-thumbnail">{thumbnail}</div>}
+        {thumbnail || (
+          <div className={`listen-thumbnail ${!thumbnailSrc ? "empty" : ""}`}>
```

Comment:
If no thumbnail is passed in props, we create the listen-thumbnail initially with an .empty css class. Once the image src is fetched the class is removed, and we get a bit of a transition.
Otherwise there's a jarring jump to the right of the track name when the  listen-thumbnail element is added.

Reply:
Sounds good!